src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -897,6 +911,7 @@ static PaError RequestStop( PaPulseAudio_Stream * stream,
     PaError ret = paNoError;
     PaPulseAudio_HostApiRepresentation *pulseaudioHostApi = stream->hostapi;
     pa_operation *pulseaudioOperation = NULL;
+    int waitLoop = 0;
```

Comment:
The `waitLoop` variable seems unused.

Reply:
@illuusio please fix this before we merge.

Reply:
Sorry this took too long but now it's fixed.